### ISSUE TITLE
Perlbrew failed to install Perl 5.14.0

### ISSUE THREAD

**Issue by p5pRT** [Perl/perl5#11358]
Migrated from [rt.perl.org#90826](https://rt-archive.perl.org/perl5/Ticket/Display.html?id=90826) (status was 'rejected')

Searchable as RT90826$

**Comment by p5pRT** [comment]
### From timo.myyra@gmail.com

[Body was too large for import. Click here for the attachment in RT](https://rt-archive.perl.org/perl5/Ticket/Attachment/WithHeaders/427126)

**Comment by p5pRT** [comment]
### From @jkeenan

AFAICT \(the size of the attachment posed difficulties\)\, your 'perlbrew'
installation successfully built Perl 5\.14\.0\, but some tests failed
during 'make test'\.  Generally the failures looked like this&#8203;:

\#\#\#
String found where operator expected at
/admin/perl5/perlbrew/build/perl\-5\.14\.0/cpan/ExtUtils\-Constant/\.\./\.\./lib/Carp\.pm
line 145\, near "$arg =~ s/'/\\\\'"
&nbsp;    \(Missing operator before '/\\\\'?\)
Global symbol "$where" requires explicit package name at
/admin/perl5/perlbrew/build/perl\-5\.14\.0/cpan/ExtUtils\-Constant/\.\./\.\./lib/Carp\.pm
line 118\.
Global symbol "$arg" requires explicit package name at
/admin/perl5/perlbrew/build/perl\-5\.14\.0/cpan/ExtUtils\-Constant/\.\./\.\./lib/Carp\.pm
line 118\.
Global symbol "$arg" requires explicit package name at
/admin/perl5/perlbrew/build/perl\-5\.14\.0/cpan/ExtUtils\-Constant/\.\./\.\./lib/Carp\.pm
line 118\.
\#\#\#
In other words\, the problems occurred in lib/Carp\.pm when called from
tests of 'dual\-life' modules\.

Apart from using perlbrew\, are you able to build and test Perl 5\.14
successfully on this machine?

Thank you very much\.
Jim Keenan

**Comment by p5pRT** [comment]
The RT System itself - Status changed from 'new' to 'open'

**Comment by p5pRT** [comment]
### From @jkeenan

On Fri Nov 25 14&#8203;:37&#8203;:52 2011\, jkeenan wrote&#8203;:
> AFAICT \(the size of the attachment posed difficulties\)\, your
> 'perlbrew'
> installation successfully built Perl 5\.14\.0\, but some tests failed
> during 'make test'\.  Generally the failures looked like this&#8203;:
> 
> \#\#\#
> String found where operator expected at
> /admin/perl5/perlbrew/build/perl\-5\.14\.0/cpan/ExtUtils\-
> Constant/\.\./\.\./lib/Carp\.pm
> line 145\, near "$arg =~ s/'/\\\\'"
>     \(Missing operator before '/\\\\'?\)
> Global symbol "$where" requires explicit package name at
> /admin/perl5/perlbrew/build/perl\-5\.14\.0/cpan/ExtUtils\-
> Constant/\.\./\.\./lib/Carp\.pm
> line 118\.
> Global symbol "$arg" requires explicit package name at
> /admin/perl5/perlbrew/build/perl\-5\.14\.0/cpan/ExtUtils\-
> Constant/\.\./\.\./lib/Carp\.pm
> line 118\.
> Global symbol "$arg" requires explicit package name at
> /admin/perl5/perlbrew/build/perl\-5\.14\.0/cpan/ExtUtils\-
> Constant/\.\./\.\./lib/Carp\.pm
> line 118\.
> \#\#\#
> In other words\, the problems occurred in lib/Carp\.pm when called from
> tests of 'dual\-life' modules\.
> 
> Apart from using perlbrew\, are you able to build and test Perl 5\.14
> successfully on this machine?
> 
> Thank you very much\.
> Jim Keenan


The OP has not responded to these questions posed in November 2011 \-\-
when I was just starting out on P5P\!

I suspect thousands of people have used perlbrew to install versions of
Perl since 5\.14\.0\, so unless we get additional data\, I recommend we
close this ticket\.  I am taking the ticket for the purpose of doing so
within 7 days\.

Thank you very much\.
Jim Keenan

**Comment by p5pRT** [comment]
### From @jkeenan

On Fri Aug 09 18&#8203;:40&#8203;:20 2013\, jkeenan wrote&#8203;:

> 
> The OP has not responded to these questions posed in November 2011 \-\-
> when I was just starting out on P5P\!
> 
> I suspect thousands of people have used perlbrew to install versions of
> Perl since 5\.14\.0\, so unless we get additional data\, I recommend we
> close this ticket\.  I am taking the ticket for the purpose of doing so
> within 7 days\.
> 

Closing as per schedule\.

**Comment by p5pRT** [comment]
@jkeenan - Status changed from 'open' to 'rejected'
